indico/modules/rb_new/client/js/modules/calendar/queryString.js

```diff
@@ -37,7 +37,12 @@ const rules = {
     mode: {
         validator: v.isIn(['days', 'weeks', 'months']),
         stateField: defaultStateField('datePicker.mode', 'days'),
-    }
+    },
+    myBookings: {
```

Comment:
I've noticed that we use camel case for some query params (`myBlockings`, `myRooms`) and snake case for some others (map lat & lng). I don't know which case is more common but I think we should be consistent.

Reply:
Yeah, true. Maybe it is a good opportunity to change it in blockings as well